test/unit/communication/test_communicator_grid.cpp

```diff
@@ -187,7 +187,7 @@ TEST_P(CommunicatorGridTest, ConstructorIncomplete) {
     EXPECT_EQ(MPI_COMM_NULL, incomplete_grid.rowCommunicator());
     EXPECT_EQ(MPI_COMM_NULL, incomplete_grid.colCommunicator());
 
-    EXPECT_EQ(-1, incomplete_grid.rank_all(coords));
+    EXPECT_EQ(-1, incomplete_grid.rankFullCommunicator(coords));
```

Comment:
Shouldn't `rankFullCommunicator` be tested with different values of coords? (applies also to other places in the code)